tests/test-cases/Group1-Docker-Commands/1-43-Docker-CP-Offline.robot

```diff
@@ -179,3 +181,69 @@ Copy a file from host to offline container, dst is a nested volume with 3 levels
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} rm -f offline
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
+
+Concurrent copy: repeat copy a small file from host to offline container several times
+    ${status}=  Get State Of Github Issue  5742
+    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-43-Docker-CP-Offline.robot needs to be updated now that Issue #5742 has been resolved
+    #${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -i --name offline -v vol1:/vol1 ${busybox}
```

Comment:
This is a serial copy - this is what was illustrating the concurrency flaw, but that's only due to a personality bug in the PR.
We also need a test where the different files are copied in parallel to the same container.
Ask @mhagen-vmware about creating a background robot process.